client/my-sites/media-library/list.jsx

```diff
@@ -152,7 +152,7 @@ export class MediaLibraryList extends React.Component {
 	};
 
 	getItemGroup = ( item ) =>
-		min( [ item.date.slice( 0, 10 ), this.props.moment( new Date() ).format( 'YYYY-MM-DD' ) ] );
+		Math.min( item.date.slice( 0, 10 ), this.props.moment( new Date() ).format( 'YYYY-MM-DD' ) );
```

Comment:
Does this array need to be spread? (...item.date.slice(0, 10))

Reply:
Nope, the date is a string, so this will grab the day from the date (`2020-07-27T01:31:39+03:00` will result in `2020-07-27`)

Reply:
Makes sense!

Reply:
I'm not sure why I didn't figure this out earlier, but it broke because we can't pass strings to `Math.min`. Discovered by @yuliyan and getting fixed in #51712.